peer_review_data/main.py

```diff
@@ -12,50 +12,47 @@
 LOGGER: Logger = getLogger(__name__)
```

Comment:
I think you want to just use `getLogger` from `logging` here. I think you're importing it through `canvasData`.

Reply:
It is.  I guess it's better not to depend on the imports of the other module.

Reply:
I had been using a logger in `canvasData`, but not any longer.  I didn't notice until I tried to add the import to `main`.  My IDE kept telling me it was unnecessary because of the import through `canvasData`.  I found I didn't use the logger there any more and I also decided I should replace the `import *` with an explicit one.

Reply:
Resolved in 5861955.